main.go

```diff
@@ -12,36 +14,70 @@ import (
 // JSON example
 // {"ar": ["joaothallis"], "elixir": ["josevalim", "eksperimental"]}
 
-func main() {
-	dir := os.Args[1]
-	homeDir, err := os.UserHomeDir()
+func getPrarFilePath() (string, error) {
+	globalFlag := flag.Bool("global", false, "When you choose to using ./config/prar.json")
+	flag.Parse()
+	if *globalFlag {
+		homeDir, err := os.UserHomeDir()
+		return homeDir + "/.config/prar.json", err
+	} else {
+		homeDir, err := os.Getwd()
+		return homeDir + "/.prar.json", err
+	}
+}
 
-	filePath := homeDir + "/.config/prar.json"
+func getProjectName() (string, error) {
+	args := flag.Args()
+	if len(args) >= 1 {
```

Comment:
If we use `--global` it will count as a arg, we need you need to verify `if args == 1 and args != global` or `if args == 2`

Reply:
just count how arg when any thing input without `-` of `-global`